CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.31.0] - 2018-07-09
+### Added
+- Added `Outbounds()` on `Dispatcher` to provide access to the configured outbounds.
+- Expose capacity option to configurator for the round-robin peer chooser.
+- Expose capacity option to configurator for the fewest pending heap peer chooser.
+- Dispatchers now log recovered handler panics via a Zap logger, if present.
+
+### Changed
+- TChannel inbounds will blackhole requests when handlers return resource
+  exhausted errors.
+- Change log level to reflect error statuses. Previously all logs were logged at
+  debug level. Errors are now logged at error level.
+- Update pin for gogo/protobuf to ^1
```

Comment:
nit: I noticed as I released this that this is not quite accurate - the pin is now actually '>=0.5, <2'